modules/websocket_bridge/include/hephaestus/utils/ws_client.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
this file should be:
`modules/websocket_bridge/include/hephaestus/websocket_bridge/utils/ws_client.h`